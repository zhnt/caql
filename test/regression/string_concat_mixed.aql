// Test string concatenation with numbers
print("Number: " + string(42));
print("Value: " + string(123));

let num = 456;
print("Result: " + string(num));
print("Count: " + string(num));

// Multiple mixed concatenations
let x = 10;
let y = 20;
print("Sum: " + string(x) + " + " + string(y) + " = " + string(x + y));
print("Product: " + string(x) + " * " + string(y) + " = " + string(x * y));
