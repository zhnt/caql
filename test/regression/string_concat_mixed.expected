Number: 42
Value: 123
Result: 456
Count: 456
Sum: 10 + 20 = 30
Product: 10 * 20 = 200

